landlab/components/radiation/radiation.py

```diff
@@ -213,76 +292,235 @@ def update(self):
         ``hour`` and uses their values in updating fields.
         """
         self._t = self._hour
-        self._radf = self._cell_values["radiation__ratio_to_flat_surface"]
-        self._Rs = self._cell_values["radiation__incoming_shortwave_flux"]
-        self._Rnet = self._cell_values["radiation__net_shortwave_flux"]
 
-        self._julian = np.floor(
-            (self._current_time - np.floor(self._current_time)) * 365.25
-        )  # Julian day
+        # Julian Day - ASCE-EWRI Task Committee Report, Jan-2005 - Eqn 25, (52)
+        self._julian = (self.current_time - np.floor(self.current_time)) * 365
```

Comment:
I modified ``self._julian`` to include the fractional part. If a user really wants to ignore the fractional part, they can do so when setting ``current_time``. Are you ok with that?

Reply:
yes this works just fine